nntrainer/tensor/manager.h

```diff
@@ -148,12 +148,13 @@ class Manager {
    *
    * @param node Graph node to extract node identifiers/info
    * @param weights_spec Specficiation for the weights
+   * @param freeze make the weight non-trainable if true
```

Comment:
minor but lets keep common terminology.
/s/freeze/trainable/g
and usage will be inverted

Reply:
Done